test/common/verify_alert_links_to_SOPs.go

```diff
@@ -0,0 +1,147 @@
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	prometheusv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+)
+
+var githubToken = os.Getenv("TOKEN_GITHUB")
+
+func TestVerifyAlertLinksInSOPs(t TestingTB, ctx *TestingContext) {
+
+	if githubToken == "" {
+		t.Skip("github token not present")
+	}
+
+	var apiLinks []string
+	testUrl := "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/README.md"
+	testResp := getSOPAlertLinkStatus(t, testUrl)
```

Comment:
-     alert link to SOP (file name)
      -  This is because test case is called like that here [](https://github.com/integr8ly/integreatly-operator/blob/master/test-cases/tests/alerts/c08b-verify-alert-links-to-sops.md). So I suppose it's right name for the file?
-     alert link in SOP (this function name)
      - Changed for "in" to "to"
-     api link (name of slice of strings)
      - Changed to sopUrls
-     sop Url (later in the code around line 60)
-     sop link (from function name validateSopLink)
      - Changed to validateSOPUrls
-     raw link (input param name to modifyLink)
      - Changed to sopUrl
-     url (if getting the link from slice later in the code)
      - Changed to sopUrl

Reply:
There's no right name. If the name is descriptive enough then it is just fine. The key thing is to use just one name for one thing across the test case code base.

Reply:
c08b-verify-alert-links-to-sops.md -> it does not mean that you have to call it alertLinkToSOP in this test. Feel free to rename it to `sopUrl` everywhere, including the file name and function name. For the sake of consistency.